students/20th/garamsong/login/login.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const loginId = document.querySelector('.loginId');
+const loginPassword = document.querySelector('.loginPassword');
+const loginBtn = document.querySelector('.loginBtn');
+
+function activeBtn(){
+let loginIdValue = loginId.value;
+  let loginPasswordValue = loginPassword.value;
+  if((loginIdValue.length) && (loginPasswordValue.length)){
+    loginBtn.classList.add('afterBtnActive');
+  } else{
+      return; 
+  }
+};
```

Comment:
```suggestion
function activeBtn() {
  const loginIdValue = loginId.value;
  const loginPasswordValue = loginPassword.value;
  
  if (loginIdValue.length && loginPasswordValue.length){
    loginBtn.classList.add('afterBtnActive');
  } else {
    return; 
  }
};
```
- 가급적 const로 변수를 선언해주는 것이 좋습니다. let과 const 차이에 대해서 알고 있는 내용을 댓글로 달아주시겠어요?
- 적절한 띄어쓰기와 indent는 가독성 측면에서 아주아주 중요한 습관입니다!
- 불필요한 소괄호가 있어 제거하였습니다.
- 현재 버튼이 활성화 되는 로직만 작성되어 있네요! 조건에 맞지 않을 경우 버튼을 비활성화 하는 로직도 추가해보시겠어요? hint: `else`

Reply:
1. const와 let 변수의 차이
- const: 값을 변경하지 않을 때 사용하는 변수 입니다. 한번 선언하면 그 뒤로 값을 바꾸지 못합니다.  재선언 재할당이 불가합니다. let이 필요한 경우가 아니면 코드의 유지보수를 위해 const를 사용합니다.   
- let: 값이 변경 가능한 변수 입니다. 한번 선언하고 그 뒤에 값을 바꿀 수 있습니다. 하지만 const와 마찬가지로 재선언은 하지 못합니다. 재할당만 가능합니다. 값을 재할당 하고 싶은땐 let을 쓰지 않고 변수이름 = "넣을값" 으로 넣을 수 있습니다. 

- 궁금한 점: 일반적인 경우에서 const를 사용한다는건 이해가 되지만 어떤 경우가 let을 써야하는 경우인지 판별이 잘 서지 않을 때가 있습니다. 아마 사용했을 때, login Id의 값은 항상 바뀌니까 let을 써야하지 않을까 잘못 생각하기도 했는데요, let을 이용하는 대표적인 경우는 어떤 것이 있을까요? 

2. 띄어쓰기와 들여쓰기
- 가독성을 위해 띄어쓰기를 한다고 하셨는데, 주로 함수 안에서 변수를 선언하는 부분과 실제 함수가 진행되는 조건문이나 반복문 사이에 띄어쓰기를 해주면 될까요? 

3. 기초적인 질문 드려 죄송합니다! 다른 부분들은 확인 후 수정했습니다. 덕분에 간단한 코드가 되었습니다 ;) 감사합니다!